packages/expo-modules-core/ExpoModulesCore.podspec

```diff
@@ -43,7 +43,9 @@ Pod::Spec.new do |s|
     s.dependency 'RCT-Folly', folly_version
   end
 
-  if !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCE'].to_i == 0 && File.exist?("ios/#{s.name}.xcframework") && Gem::Version.new(Pod::VERSION) >= Gem::Version.new('1.10.0')
+  ex_updates_native_debug = ENV['EX_UPDATES_NATIVE_DEBUG'] == '1'
+
+  if !ex_updates_native_debug && !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCE'].to_i == 0 && File.exist?("ios/#{s.name}.xcframework") && Gem::Version.new(Pod::VERSION) >= Gem::Version.new('1.10.0')
```

Comment:
Why is that needed in `expo-modules-core`? Anyway, you can revert that — we don't prebuild this package.

Reply:
If we are not prebuilding the package, maybe just remove this clause entirely and just include source files?

Reply:
Actually we didn't make the decision what to do with prebuilds yet. Kudo is still figuring out if there is an option to prebuild them even if Swift code is there.